perf/perf_duplicate_probe.py

```diff
@@ -45,14 +45,16 @@ def setUp(self):
                 self.cancel('%s is needed for the test to be run' % package)
 
     def test_probe(self):
-        output = process.run("perf probe select_task_rq_fair:15", sudo=True)
-        if 'select_task_rq_fair' in output.stderr.decode("utf-8") and 'select_task_rq_fair_' in output.stderr.decode("utf-8"):
-            fail_flag = 1
+        output = process.run("perf probe select_task_rq_fair:0", sudo=True)
+        output1 = output.stderr.decode("utf-8")
```

Comment:
can you name this variable better or just replace the output if its not used elsewhere?

Reply:
ok